src/hashtable.py

```diff
@@ -65,15 +65,13 @@ def insert(self, key, value):
     def remove(self, key):
         '''
         Remove the value stored with the given key.
-
         Print a warning if the key is not found.
-
         Fill this in.
         '''
         index = self._hash_djb2(key)
 
         if self.storage[index] is not None:
-            self.st
+            self.storage[index] = None
```

Comment:
you should check the key of the pair stored here to make sure we're deleting the right thing. Also need to check if there is anything linked on and make sure not to delete anything extra.